package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "@flowfuse/nr-launcher",
-    "version": "2.2.2",
+    "version": "2.2.3",
```

```suggestion
    "version": "2.2.2",
```

We dont bump version. This is dont when the release is cut.